functional-tests/bdd-tests/CRS_review_federal_reports.feature

```diff
@@ -0,0 +1,11 @@
+﻿Feature: CRS_review_federal_reports
+    As a logged in business user
+    I want to review the Cannabis Retail Store federal reports
+
+Scenario: Review CRS Federal Reports
+    Given the CRS application has been approved
+    And I am logged in to the dashboard as a private corporation
+    And I click on the Licences tab
+    And the licence fee has been paid
+    And I click on the Review Federal Reports link
+    Then the monthly reports will be available
```

Comment:
This is good as a start, but please think about the timing of this feature and whether or not that's something to worry about. New monthly reports are generated on the 1st of each month, they are bundled up and sent as a batch file on the 15th of each month. Statuses are updated when the report is created, updated by the user, submitted by the user and bundled up.